### PR TITLE
Create reviews

### PR DESCRIPTION
**Pull request by noahmilstein**

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+class ReviewsController < ApplicationController
+  before_action :authorize_user, except: [:index]
+
+  def create
+    @review = Review.new(review_params)
+    @movie = Movie.find(params[:movie_id])
+    @ratings = Movie::RATINGS
```
Comment (jennceng) [inline app/controllers/reviews_controller.rb]: I'm not seeing where Movie::RATINGS is defined?

File: app/models/review.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class Review < ActiveRecord::Base
+  validates :title, presence: true
+  validates :rating, numericality: { integer_only: true }
+  validates :body, presence: true, length: { minimum: 50, message:
+    "Body must be at least 50 characters" }
+
+  validate :rating_range
+  def rating_range
+    unless (1..5).cover?(self.rating.to_i)
```
Comment (jennceng) [inline app/models/review.rb]: self here is implicit, you can remove it

File: app/controllers/reviews_controller.rb (hunk 1 above)
Comment (noahmilstein) [inline app/controllers/reviews_controller.rb]: Movie::RATINGS is in the Movie model

File: app/models/review.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      errors.add(:rating_range, "Rating must be 1 - 5")
+    end
+  end
```
Comment (jennceng) [inline app/models/review.rb]: I'm not sure what the purpose of rating range is?  Why not just add this to the rating validation?

Comment (noahmilstein) [inline app/models/review.rb:12]: ActiveRecord would not validate inclusion so we had to create a custom method to made the validation pass.

File: spec/features/08_create_reviews_spec.rb [hunk 4]
```diff
@@ -1,12 +1,46 @@
 require "rails_helper"
 
-feature "Authenticated user" do
+feature "Create reviews user" do
   let!(:user) { FactoryGirl.create(:user) }
   let!(:movie1) { FactoryGirl.create(:movie) }
   let!(:movie2) { FactoryGirl.create(:movie) }
+  let!(:review) { FactoryGirl.create(:review, movie_id: movie1.id) }
```
Comment (noahmilstein) [inline spec/features/08_create_reviews_spec.rb]: Can do!

File: app/models/review.rb (hunk 2 above)
Comment (noahmilstein) [reply app/models/review.rb]: Good to know!

**Pull request merged**